### PR TITLE
FIX: 0 amounts in custom receive would result in undefined

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/75ckd0cnrp0cqg1pa6g1gudg5r?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9047)

File: screen/receive/details.js [hunk 1]
```diff
@@ -447,7 +447,15 @@ const ReceiveDetails = () => {
         }
         break;
     }
-    setBip21encoded(DeeplinkSchemaMatch.bip21encode(address, { amount, label: customLabel }));
+    const properties = {};
```
Comment (Overtorment) [inline screen/receive/details.js:450]: so the problem is in extra spaces?
why not move this logic inside of `DeeplinkSchemaMatch.bip21encode`?
and add a unit test for that as well

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: class/deeplink-schema-match.js [hunk 2]
```diff
@@ -446,7 +446,16 @@ class DeeplinkSchemaMatch {
   }
 
   static bip21encode() {
-    return bip21.encode.apply(bip21, arguments);
+    const argumentsArray = Array.from(arguments);
```
Comment (Overtorment) [inline class/deeplink-schema-match.js:449]: nice! wheres the unit test for that?

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/v3dew2yfn6ywuz7uqrtuj6q338?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9049)

File: screen/receive/details.js (hunk 1 above)
Comment (marcosrdz) [reply screen/receive/details.js]: Done

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/g288u7k6e2522h8yu3y2xrjbvw?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9051)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**